ops/helmfiles/config/holesky.yaml

```diff
@@ -18,7 +18,7 @@ collator:
     repository: mangatasolutions/rollup-node
     tag: 3a07824d8b8a6ad027424629b814a5edb61654cf
   disableChainspecGeneration: true
-  customChainspecUrl: https://raw.githubusercontent.com/mangata-finance/chainspec/master/rollup-holesky-3a07824d8b8a6ad027424629b814a5edb61654cf-raw.json
+  customChainspecUrl: https://raw.githubusercontent.com/mangata-finance/chainspec/e6dbb96641a79a8d77c2ce79c20c1ed598958f00/rollup-holesky-3a07824d8b8a6ad027424629b814a5edb61654cf-raw.json
```

Comment:
This chainspec it seems does not have the salt... I guess that is to prevent redeployment of the holesky env... but lets keep in mind to use a salted spec in the next version of the testnet...